### PR TITLE
Relation with other software

### PR DESCRIPTION
**Pull request by rolandlo**
With this PR I address a FAQ about the relation between Xournal++ and Xournal, while comparing Xournal++ with other software like Xournal++ mobile, Mr Writer and Joplin as well.

### CODE REVIEW

File: docs/community/otherSoftware.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+### Known issues/bugs/inconveniences of Xournal, resolved in Xournal++
+
+TODO
+
+### Performance comparision between Xournal++ and Xournal
+
+- On most systems Xournal++ has significantly reduced memory usage compared to Xournal
+- Xournal++ uses PDF caching for faster display of PDF pages
+- Xournal++ has code to detect memory leaks, which Xournal does not
+- Xournal++'s .xopp files are usually lower in file size thanks to reducing unnessarily high precision of stroke coordinates
```
Comment (Technius) [inline docs/community/otherSoftware.md:61]: Do you know if these claims are still true?

**Review by Technius — COMMENTED** [review]
Looks fine to me. `otherSoftware.md` should be renamed to `other-software.md` or else the URL will be weird.

File: docs/community/otherSoftware.md (hunk 1 above)
Comment (rolandlo) [reply docs/community/otherSoftware.md:61]: I am not sure. I think it would be good to make some fresh performance measurements comparing the two apps.

Comment (rolandlo) [reply docs/community/otherSoftware.md]: Hmm, these claims seem on thin ice. I rather delete them. In particular the last one is plain false. Xournal only stores two decimal places. Moreover I have the impression that something must be wrong with pdf caching in Xournal++. After switching from a large pdf to a new document I couldn't see the memory consumption drop to a reasonable level (comparable to after startup).

Comment (rolandlo) [reply docs/community/otherSoftware.md]: I have removed the unbased claims and will merge now.

**Pull request merged**